workspaces/grafana/plugins/grafana/docs/embed-dashboards-on-page.md

```diff
@@ -0,0 +1,55 @@
+# Embed dashboards on a component page
+
+Adding the `EntityOverviewDashboardViewer` component to an entity's page will embed an "overview" dashboard defined by the entity itself.
+
+```ts
+// packages/app/src/components/catalog/EntityPage.tsx
+
+import { EntityGrafanaDashboardsCard } from '@k-phoen/backstage-plugin-grafana';
```

Comment:
I noticed this issue in the original repo. Think this import needs to match the grid item you actually use.

```suggestion
import { EntityOverviewDashboardViewer } from '@k-phoen/backstage-plugin-grafana';
```

Reply:
Indeed. There were a couple leftovers pointing to the old repo in the documentation.

I've updated those references and use the correct ones now. In one case I also needed to update the component name